faculty-affiliations.csv

```diff
@@ -3206,12 +3208,14 @@ Matthias Hollick , TU Darmstadt
 Max Mühlhäuser , TU Darmstadt
 Melanie Volkamer , TU Darmstadt
 Michael Goesele , TU Darmstadt
+Michael Pradel , TU Darmstadt
 Michael Waidner , TU Darmstadt
 Mira Mezini , TU Darmstadt
 Neeraj Suri , TU Darmstadt
 Oskar von Stryk , TU Darmstadt
 Patrick Eugster , TU Darmstadt
 Patrick Th. Eugster , TU Darmstadt
+Ralf Steinmetz , TU Darmstadt
```

Comment:
According to the dept. page, he has a courtesy appointment -- only faculty who can advise PhD students in the CS dept. are listed.